articles/the-future-of-device-management-ep6.md

```diff
@@ -0,0 +1,33 @@
+# Future of device management
+## EP 6: Whitney Champion: Scaling infrastructure automation with open source tools
+
+<iframe allow="autoplay *; encrypted-media *; fullscreen *; clipboard-write" frameborder="0" height="175" style="width:100%;max-width:660px;overflow:hidden;background:transparent;" sandbox="allow-forms allow-popups allow-same-origin allow-scripts allow-storage-access-by-user-activation allow-top-navigation-by-user-activation" src="https://embed.podcasts.apple.com/us/podcast/ep-6-whitney-champion-scaling-infrastructure-automation/id1641183838?i=1000580448335"></iframe>
+
+Listen to the episode on [Apple](https://podcasts.apple.com/us/podcast/ep-6-whitney-champion-scaling-infrastructure-automation/id1641183838?i=1000580448335), [Spotify](https://open.spotify.com/episode/4ywwvk1vSj0ENhrAg4k1Ii?si=w6dg7PlsQ0CqTQQXQb7RTQ), or [PodBean](https://www.podbean.com/ew/pb-z3dhr-12ced49)
+
+Whitney is a co-founder and the lead architect at Recon InfoSec. She has extensive experience in large-scale infrastructure automation and orchestration in both the private and public sector. She builds and maintains the security stack that powers Recon’s MDR services and their network defense range, which powers their NDR training and OpenSOC.io. Listen in to today's episode to learn how Whitney got her start in cybersecurity, and how open source tools can help lay the foundation for better security infrastructure.
```

Comment:
```suggestion
Whitney Champion is a co-founder and the lead architect at Recon InfoSec. She has extensive experience in large-scale infrastructure automation and orchestration in both the private and public sector. She builds and maintains the security stack that powers Recon’s MDR services and their network defense range, which supports their NDR training and OpenSOC.io. In this episode, Whitney discusses how she got her start in cybersecurity and how open source tools can help lay the foundation for better security infrastructure.
```
Not sure if "support" is the best word, but I was trying to add some variety. Feel free to swap it for something else or leave it as "powers" if you feel that's the most accurate.